### DISCUSSION TITLE
EAP-TLS session setup stuck

### DISCUSSION THREAD

**Discussion by denpa290** [strongswan/strongswan#2276] [General]
Hello, I am trying to switch IKEv2 authentication of Windows 10 clients from machine certificate authentication to EAP-TLS authentication with user certificates. I followed the docs here: https://docs.strongswan.org/docs/5.9/interop/windowsClients.html#_using_x_509_user_certificates
The server certificate I am using is valid and anyway it works with machine certificate authentication, so everything should be trusted. 
When I try the connection to strongswan server (5.9.10) I send the client a EAP/SUCC message, but then nothing happens anymore, On windows side it states "Internal Authentication error". Do somebody has any hint?

Thanks everybody

[eap-tls-conf.txt](https://github.com/user-attachments/files/15564942/eap-tls-conf.txt)
[charon.log](https://github.com/user-attachments/files/15565778/charon.log)

**Comment by tobiasbrunner** [comment]
Yes, that's due to a regression in the _eap-tls_ plugin introduced with 5.9.10 (see #1613, was fixed with 3d0d3f5d028a0c630f89cb4fec5b2cd5364f568b that was included in 5.9.11).
